cpython-windows/build.py

```diff
@@ -1388,14 +1384,15 @@ def build_cpython(
     python_entry_name: str,
     target_triple: str,
     arch: str,
-    profile,
+    build_options: str,
     msvc_version: str,
     windows_sdk_version: str,
     openssl_archive,
     libffi_archive,
     openssl_entry: str,
 ) -> pathlib.Path:
-    pgo = profile == "pgo"
+    parsed_build_options = set(build_options.split("+"))
```

Comment:
Was this wrong before? Or was it just never called with `pgo+lto`?

Reply:
Ah there are only "pgo" builds on Windows. See https://github.com/indygreg/python-build-standalone/pull/326/files#r1777582385

Reply:
Yeah this seems like a good change.